docs/contribute/new-provider.rst

```diff
@@ -0,0 +1,73 @@
+.. _new-provider:
+
+New provider
+============
+
+The actual implementation gives two providers: :doc:`grid5000` and
+:doc:`vbox`. If you want to support another testbed, then implementing
+a new one is easy as 500 lines of Python code.
```

Comment:
* vbox is less than 150 LOC
* g5k is 450 LOC
* (estimation) openstack should be less than 300 LOC
* (estimation) chameleon should be approx 200 LOC

i think 500 LOC is a bit too much :)